src/middleware/header.js

```diff
@@ -180,8 +180,8 @@ module.exports = function (middleware) {
 				templateValues.browserTitle = results.browserTitle;
 				templateValues.navigation = results.navigation;
 				templateValues.unreadCount = unreadCount;
-				templateValues.metaTags = results.tags.meta;
-				templateValues.linkTags = results.tags.link;
+				templateValues.metaTags = data._header.tags.meta;
+				templateValues.linkTags = data._header.tags.link;
```

Comment:
Instead of being retrieved up above (the since-removed line `110`), the header data is parsed earlier inside `processRender`, and is used in both cold loads and ajaxify. It is placed in the temporary `options._header` property.